background/get-addon-settings.js

```diff
@@ -19,45 +45,60 @@ chrome.storage.sync.get(["addonSettings", "addonsEnabled"], ({ addonSettings = {
         delete settings.editorCount;
         addonsEnabled["block-count"] = true;
       }
+      if (addonId === "editor-dark-mode") {
+        // Transition v1.22 to v1.23
+        // TurboWarp Dark preset changes:
+        updatePresetIfMatching(
+          manifest.presets.find((p) => p.id === "tw-dark"),
+          settings,
+          {
```

Comment:
Listing the present settings here is probably a bad thing to do. You can get the data from the manifest.

Reply:
Old presets are not available in the manifest.

Reply:
Oh, I thought this was the new one